roundware/api2/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
```

Comment:
Please add the standard licensing lines that should be in all code files (including `__init.py__` above)
```
# Roundware Server is released under the GNU Affero General Public License v3.
# See COPYRIGHT.txt, AUTHORS.txt, and LICENSE.txt in the project root directory.
```